components/admin_console/admin_definition.jsx

```diff
@@ -1322,15 +1471,81 @@ const AdminDefinition = {
             ],
             schema: {
                 id: 'SessionLengths',
-                component: SessionLengthSettings,
```

Comment:
Is this component not used anywhere now?, I feel like im issing something

Reply:
Looks like that was a bad conflict resolution. Re-added.